packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2157,19 +2156,6 @@ describe("Runtime", () => {
 				// but the "missingDataStore" is requested/aliased, it fails to because for summarizer the fetched snapshot could
 				// not be ahead of the base snapshot as that means that a snapshot is missing and the summarizer is not up to date.
 				containerContext.storage.getSnapshot = async (snapshotFetchOptions) => {
-					snapshotTree.trees[".channels"].trees.missingDataStore = {
-						blobs: {
-							".component": "bARC6dCXlcrPxQHw3PeROtmKc",
-						},
-						trees: {
-							".channels": {
-								blobs: {},
-								trees: {
-									root: { blobs: {}, trees: {} },
-								},
-							},
-						},
-					};
```

Comment:
Could you explain why this was deleted? I can't tell from an immediate glance.

Reply:
Because after the decision that we will return full tree on initial call which is snapshotWithContents.snapshotTree will be a full tree, now we don't need to add this tree anymore on grouped snapshot fetch call as snapshotWithContents.snapshotTree  would already have this tree.